include/seqan/arg_parse/arg_parse_version_check.h

```diff
@@ -122,6 +123,11 @@ struct VersionCheck
         errorStream(errorStream)
     {
         std::smatch versionMatch;
+#if defined(NDEBUG) || defined(SEQAN_TEST_VERSION_CHECK_)
+        _timestamp_filename = _path + "/" + _name + "_release.timestamp";
+#else
+        _timestamp_filename = _path + "/" + _name + "_debug.timestamp";
```

Comment:
maybe call it `_dev.timestamp` and `_usr.timestamp` to indicate that the one is meant for developers and the other for users.